build/makelib/golang.mk

```diff
@@ -156,6 +156,14 @@ go.test: go.vet go.fmt $(CGO_PREREQS)
 #	@$(GOHOST) test -v -i $(GO_PKG_FLAGS) $(GO_NONSTATIC_FLAGS) $(GO_ALL_PACKAGES)
 	@$(GOHOST) test -cover $(GO_PKG_FLAGS) $(GO_NONSTATIC_FLAGS) $(GO_ALL_PACKAGES)
 
+.PHONY: go.lint
+go.lint:
```

Comment:
cool, let's get a github issue opened about fixing the lint errors.  once that's done we can add the lint target to the default makefile path so it runs on every build.